packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue

```diff
@@ -9,20 +9,39 @@
     <span class="api-label api-label-deprecated" v-if="access == 'deprecated'">
       <i class="fa fa-fire-extinguisher"></i> Deprecated
     </span>
+    <span class="api-label api-label-ie" v-if="access == 'ie'">
```

Comment:
I guess you're following the old pattern here but it would be great if we could use `===` and also check equality with constants rather than the string literals here

Reply:
about constants seems it not possible to `labelType` inside html, it undefined there, so or to use it inside `data` or just like string

Reply:
I think it would be worth exploring one of [these solutions](https://coderethinked.com/3-different-ways-to-access-constants-in-a-vue-template/). I like the "Create private variables in created() hook" one the best.

This is definitely a point in favor of React over Vue 😄